doc/manual/manual.tex

```diff
@@ -10872,6 +10872,43 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
 The parameter file and initial temperature model for this benchmark can be found at \url{benchmarks/yamauchi_takei_2016_anelasticity/yamauchi_takei_2016_anelasticity.prm} and \url{benchmarks/yamauchi_takei_2016_anelasticity/anelasticity_temperature.cc}. Code to recreate  Fig.~\ref{fig:anelasticity}  is provided in \url{benchmarks/yamauchi_takei_2016_anelasticity/plot_output}.
 
 
+\subsubsection{Brittle thrust wedges benchmark}
+\label{sec:benchmark-brittle-thrust-wedge}
+
+\textit{This section was contributed by Sibiao Liu, Stephanie Sparks, John Naliboff, and Cedric Thieulot.}
+
+Buiter et al. (2016, \cite{buiter16}) organized new comparison experiments with analogue and numerical models to investigate brittle thrust wedge behavior. The benchmark here aims to verify that the wedge models using  \aspect{} follows other numerical results and the analytical wedge theory showing in this paper. In particular, input files (\url{benchmarks/ buiter_et_al_2016_jsg}) are provided for reproducing the numerical simulations of stable wedge experiment 1 and unstable wedge experiment 2 with same model setups.
+
+A number of model sets of prescribed material behaviorl are required to simulate the brittle thrust formation. For example, although the material in the numerical model has a visco-plastic rheology, it performs plastic yielding at the beginning of shortening due to the non-viscos sand. The plastic strain-weakening behavior, with the internal angle of friction reducing between total finite strain invariant values of 0.5 and 1.0, is prescribed to mimic the softening from peak to dynamic stable strength correlates with sand dilation. 
```

Comment:
```suggestion
A number of model sets of prescribed material behavior are required to simulate the brittle thrust formation. For example, although the material in the numerical model has a visco-plastic rheology, it performs plastic yielding at the beginning of shortening due to the non-viscous sand. We prescribe plastic strain-weakening behavior, with the internal angle of friction reducing between total finite strain invariant values of 0.5 and 1.0, to mimic the softening from peak to dynamic stable strength correlates with sand dilation. 
```

Reply:
"We prescribe plastic strain-weakening behavior, with the internal angle of friction reducing between total finite strain invariant values of 0.5 and 1.0, to mimic the softening from peak to dynamic stable strength correlates with sand dilation. " -> "We prescribe plastic strain-weakening behavior, with the internal angle of friction **diminishing** between total finite strain invariant values of 0.5 and 1.0, to mimic the softening from peak to dynamic stable strength **which** correlates with sand dilation.